### PR TITLE
refactor stability main function

### PR DESCRIPTION
**Pull request by weekface**

### CODE REVIEW

File: tests/cmd/stability/main.go [hunk 1]
```diff
@@ -44,441 +36,186 @@ func main() {
...
-	var physicalNode string
-	for _, nodes := range cfg.Nodes {
-		for _, node := range nodes.Nodes {
-			if node == faultNode {
-				physicalNode = nodes.PhysicalNode
-			}
-		}
+	fta.StartNodeOrDie(physicalNode, node)
+	for _, cluster := range allClusters {
+		oa.CheckTidbClusterStatusOrDie(cluster)
```
Comment (xiaojingchen) [inline tests/cmd/stability/main.go]: there need to check recover

**Comment by xiaojingchen** [comment]
@tennix  @onlymellb PTAL

File: tests/actions.go [hunk 2]
```diff
@@ -67,42 +67,53 @@ func NewOperatorActions(cli versioned.Interface, kubeCli kubernetes.Interface, c
...
+	CheckScheduledBackup(info *TidbClusterConfig) error
+	DeployIncrementalBackup(from *TidbClusterConfig, to *TidbClusterConfig) error
+	CheckIncrementalBackup(info *TidbClusterConfig) error
+	Restore(from *TidbClusterConfig, to *TidbClusterConfig) error
+	CheckRestore(from *TidbClusterConfig, to *TidbClusterConfig) error
+	ForceDeploy(info *TidbClusterConfig) error
+	CreateSecret(info *TidbClusterConfig) error
+	GetPodUIDMap(info *TidbClusterConfig) (map[string]types.UID, error)
+	GetNodeMap(info *TidbClusterConfig, component string) (map[string][]string, error)
+	getBackupDir(info *TidbClusterConfig) ([]string, error)
```
Comment (zyguan) [inline tests/actions.go]: Others cannot implement the interface due to this private method.

Comment (weekface) [inline tests/actions.go]:
```suggestion
```

**Pull request merged**